invokeai/frontend/web/src/features/nodes/components/ProgressImageNode.tsx

```diff
@@ -1,18 +1,32 @@
 import { Flex, Image } from '@chakra-ui/react';
-import { NodeProps } from 'reactflow';
-import { InvocationValue } from '../types/types';
-
-import { useAppSelector } from 'app/store/storeHooks';
+import { NodeProps } from 'react-flow-renderer';
+import { useSelector, useDispatch } from 'react-redux';
+import { RootState } from 'app/store/store';
 import { IAINoContentFallback } from 'common/components/IAIImageFallback';
-import { memo } from 'react';
+import { memo, useEffect, useState } from 'react';
 import IAINodeHeader from './IAINode/IAINodeHeader';
-import IAINodeResizer from './IAINode/IAINodeResizer';
+import IAINodeResizer, { OnResize } from './IAINode/IAINodeResizer';
 import NodeWrapper from './NodeWrapper';
+import { nodeSizeChanged } from '../store/nodesSlice';
 
-const ProgressImageNode = (props: NodeProps<InvocationValue>) => {
-  const progressImage = useAppSelector((state) => state.system.progressImage);
+const ProgressImageNode = (props: NodeProps) => {
+  const progressImage = useSelector(
+    (state: RootState) => state.system.progressImage
+  );
+  const nodeSize = useSelector((state: RootState) => state.nodes.nodeSize);
+  const dispatch = useDispatch();
   const { selected } = props;
 
+  useEffect(() => {
+    localStorage.setItem('nodeSize', JSON.stringify(nodeSize));
```

Comment:
If nodeSize is in the Slice, it should automatically be stored in local storage unless it is being explicitly being denied somewhere. In that case, this would not be necessary?

Reply:
yes sorry that was left over from my first attempt to use local storage, I removed it in [remove unused code](https://github.com/invoke-ai/InvokeAI/pull/3833/commits/d1e857438bd0c1819fffff7881c1d330cde58f84)